Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Person 1:");
+            getPersonData();
+            Console.WriteLine("Person 2:");
+            getPersonData();
+
+        }
+        public static void getPersonData()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'getPersonData' to match pascal case naming rules, consider using 'GetPersonData'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887979)